src/index.ts

```diff
@@ -109,6 +114,13 @@ export function subst(template: string, params: ParamMap): string {
   return renderedPath;
 }
 
+/**
+ * Creates the path and path :params
+ * @param {String} template a string that contains :params.
+ * @param {Object} params on object with keys that correspond to the params in the template.
+ *
+ * @returns {String, String} Rendering path and rendering :params.
+ */
```

Comment:
This is an internal function that does not need a jsdoc string. Can you please remove it?